osds/verify.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Copyright 2019 The OpenSDS Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if ! [ -f "./bin/gometalinter" ]; then echo "Download The files";
+	curl -L https://git.io/vp6lP | sh
+fi
+export PATH=${PATH}:${GOPATH}/bin:${GOPATH}/src/github.com/opensds/opensds/osds/bin
+gometalinter --disable-all --enable=gofmt --enable=misspell --enable=golint --exclude=vendor --exclude=test ./../... > golintoutput.txt
+if [ $? != 0 ]; then
```

Comment:
each of the sub task scripts should return the status (pass/fail with details) to the outside caller, right ?

Reply:
if [ $? != 0 ]; then --> this check will return the status
but not to fail build, not returning error on failure... otherwise for all PRs, build will fail as many golint warnings are there